cleanup-passes/package-json.ts

```diff
@@ -12,146 +12,67 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-'use strict';
-
-import * as fs from 'fs';
+import * as fse from 'fs-extra';
```

Comment:
I think we can just import this as `fs`

Reply:
I much prefer giving it a distinct name, since actually all the APIs I use are *not* part of the standard fs, and I've seen `fse` in some other repos.